libs/design/src/atoms/form/radioset/radioset.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { DaffRadioSetComponent } from './radioset.component';
+
+
+
+@NgModule({
+  exports: [
+    DaffRadioSetComponent
+  ],
+  declarations: [
+    DaffRadioSetComponent
+  ],
+  imports: [
+    CommonModule
+  ]
+})
+export class DaffRadioSetModule { }
```

Comment:
I don't think this should be a module. I think this should probably come with Radio by default.

Reply:
Removed in 7eff94fc15e2ee0f2adf31c5de274475d7e0c73d